redbot/core/utils/chat_formatting.py

```diff
@@ -287,18 +291,20 @@ def strikethrough(text: str) -> str:
     ----------
     text : str
         The text to be marked up.
+    escape_formatting : `bool`, optional
+        Set to :code:`False` to not escape markdown formatting in the text.
 
     Returns
     -------
     str
         The marked up text.
 
     """
-    text = escape(text, formatting=True)
+    text = escape(text, formatting=escape_formatting)
     return "~~{}~~".format(text)
 
 
-def underline(text: str) -> str:
+def underline(text: str, escape_formatting: bool = True) -> str:
     """Get the given text with an underline.
 
     Note: This escapes text prior to underlining
```

Comment:
This needs an update, maybe:
```suggestion
    Note: By default, this function will escape ``text`` prior to underlining.
```